pyqir/src/qis.rs

```diff
@@ -12,309 +12,408 @@ use llvm_sys::{
 use pyo3::prelude::*;
 use qirlib::qis;
 
-/// An instruction builder that generates instructions from the basic quantum instruction set.
+/// Inserts a barrier instruction
 ///
-/// :param Builder builder: The underlying builder used to build QIS instructions.
-#[pyclass]
+/// :param builder: The underlying builder used to build QIS instructions.
```

Comment:
"Underlying builder" sounds odd since it's not being stored in a class anymore, can you use the same wording as the rt functions?